manager/integration/tests/test_system_backup_restore.py

```diff
@@ -199,7 +200,7 @@ def test_system_backup_and_restore_volume_with_backingimage(client, core_api, vo
     check_backing_image_disk_map_status(client, BACKING_IMAGE_NAME, 3, "ready")
 
     restored_volume = client.by_id_volume(volume_name)
-    wait_for_volume_restoration_completed(client, volume_name)
+    wait_for_volume_creation(client, volume_name)
```

Comment:
Does it mean we don't know whether the volume is restored?
Cause we don't check if the data is correct after restoring the system backup.

Reply:
Yes, you're right. The test case is missing a data integrity check. I'll create a separate ticket for that. Thank you!

Reply:
Using https://github.com/longhorn/longhorn/issues/10052 to track.